tests/unit/customizations/s3/test_utils.py

```diff
@@ -386,14 +386,18 @@ def test_get_file_stat(self):
             self.assertEqual(time.mktime(update_time.timetuple()), epoch_now)
 
     def test_get_file_stat_error_message(self):
+        with mock.patch('os.stat', mock.Mock(side_effect=IOError('msg'))):
+            with self.assertRaisesRegexp(ValueError, 'myfilename\.txt'):
+                get_file_stat('myfilename.txt')
+
+    def test_get_file_stat_returns_epoch_on_invalid_timestamp(self):
         patch_attribute = 'awscli.customizations.s3.utils.datetime'
         with mock.patch(patch_attribute) as f:
             with mock.patch('os.stat'):
```

Comment:
Right now it's there so we don't have to create a file. We could certainly create one, but it provides no benefit and takes up more lines.